src/pages/workspace/distanceRates/PolicyDistanceRateTaxReclaimableOnEditPage.tsx

```diff
@@ -0,0 +1,112 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React, {useCallback} from 'react';
+import {Keyboard} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx';
+import AmountForm from '@components/AmountForm';
+import FormProvider from '@components/Form/FormProvider';
+import InputWrapperWithRef from '@components/Form/InputWrapper';
+import type {FormOnyxValues} from '@components/Form/types';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import useAutoFocusInput from '@hooks/useAutoFocusInput';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import {validateTaxClaimableValue} from '@libs/PolicyDistanceRatesUtils';
+import type {SettingsNavigatorParamList} from '@navigation/types';
+import AccessOrNotFoundWrapper from '@pages/workspace/AccessOrNotFoundWrapper';
+import * as Policy from '@userActions/Policy';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type SCREENS from '@src/SCREENS';
+import INPUT_IDS from '@src/types/form/PolicyDistanceRateTaxReclaimableOnEditForm';
+import type * as OnyxTypes from '@src/types/onyx';
+
+type PolicyDistanceRateTaxReclaimableOnEditPageOnyxProps = {
+    /** Policy details */
+    policy: OnyxEntry<OnyxTypes.Policy>;
+};
+
+type PolicyDistanceRateTaxReclaimableOnEditPageProps = PolicyDistanceRateTaxReclaimableOnEditPageOnyxProps &
+    StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.DISTANCE_RATE_TAX_RECLAIMABLE_ON_EDIT>;
+
+function PolicyDistanceRateTaxReclaimableOnEditPage({policy, route}: PolicyDistanceRateTaxReclaimableOnEditPageProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const {inputCallbackRef} = useAutoFocusInput();
+
+    const policyID = route.params.policyID;
+    const rateID = route.params.rateID;
+    const customUnits = policy?.customUnits ?? {};
+    const customUnit = customUnits[Object.keys(customUnits)[0]];
+    const rate = customUnit.rates[rateID];
+    const currency = rate.currency ?? CONST.CURRENCY.USD;
+    const currentTaxReclaimableOnValue = rate.attributes?.taxClaimablePercentage && rate.rate ? (rate.attributes.taxClaimablePercentage * rate.rate) / 100 : '';
+    const submitTaxReclaimableOn = (values: FormOnyxValues<typeof ONYXKEYS.FORMS.POLICY_DISTANCE_RATE_TAX_RECLAIMABLE_ON_EDIT_FORM>) => {
+        Policy.updatePolicyDistanceRateValue(policyID, customUnit, [
```

Comment:
I think we should make it 1:1:1 since this function is for updating the rate value. I'll add a new API, which we can use with similar parameters.

Reply:
I added 2 function `UpdateDistanceTaxClaimableValue ` && `UpdateDistanceTaxRate`, its use the same parameters and the same API with function `updatePolicyDistanceRateValue `
When the new API is done, please let me know so I can fix here